pages/instructorIssues/instructorIssues.ejs

```diff
@@ -29,112 +30,156 @@
+                </a>
+              </small>
+            </div>
+            <button class="btn btn-sm btn-danger ml-auto" data-toggle="modal" data-target="#closeAllIssuesModal">
+              <i class="fa fa-times" aria-hidden="true"></i> Close all issues
+            </button>
+          </div>
+          <form name="query-form" method="GET">
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <button class="btn btn-med-light dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Filters</button>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="<%= commonQueries.allOpenQuery %>">Open issues</a>
+                  <a class="dropdown-item" href="<%= commonQueries.allClosedQuery %>">Closed issues</a>
+                  <a class="dropdown-item" href="<%= commonQueries.allManuallyReportedQuery %>">Manually-reported issues</a>
+                </div>
+              </div>
+              <input type="text" class="form-control" name="q" value="<%= filterQuery %>">
+              <div class="input-group-append">
+                <a class="btn btn-med-light" href="<%= urlPrefix %>/issues" title="Clear filters">
+                  <i class="fa fa-times" aria-hidden="true"></i>
+                </a>
+                <button class="btn btn-med-light" type="button" title="Show filter help" data-toggle="modal" data-target="#filterHelpModal">
+                  <i class="fa fa-question-circle" aria-hidden="true"></i>
+                </button>
               </div>
             </div>
+          </form>
+        </div>
+
+        <%
+        var getFormattedMessage = function(row) {
+          if (row.student_message == null || row.student_message.length == 0) {
+            return '—';
+          }
+
+          var msg = row.student_message;
+          if (row.manually_reported) {
+            msg = '"' + msg + '"';
+          }
+          return msg;
+        };
+        %>
+
+        <div class="list-group list-group-flush">
+          <% rows.forEach(row => { %>
+          <div class="list-group-item issue-list-item d-flex flex-row align-items-center">
+            <div style="min-width: 0;">
+              <% if (row.open) { %>
+              <i class="fa fa-exclamation-circle text-danger issue-status-icon"></i>
+              <% } else { %>
+              <i class="fa fa-check-circle text-success issue-status-icon"></i>
+              <% } %>
+              <a class="d-block" href="<%= urlPrefix %>/question/<%= row.question_id %>/?variant_id=<%= row.variant_id %>">
+                <%= row.question_qid ? (row.question_qid + ': ') : '' %><%= getFormattedMessage(row) %>
+              </a>
+              <small class="text-muted mr-2">#<%= row.display_id %> reported <span title="<%= row.formatted_date %>"><%= row.relative_date %></span> by <%= row.user_uid || '-' %></small>
```

Comment:
I'm thinking that we should maybe abandon `display_id` and just use the internal database `id` instead. Thoughts?

Reply:
Yep. The random ordering of the display ID is a bit confusing, imo.